src/components/icon/assets/node_slave.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Are you sure it's not already added? As "node" or something else? I'm almost sure we are using this already, you can cycle through existing ones in storybook.

Reply:
Let me recheck! Maybe I missed that this already exists

Reply:
Yeap! It existed. I removed it! Thanks @Azein